packages/cli/src/stats/__tests__/integration/compute.test.ts

```diff
@@ -0,0 +1,106 @@
+import * as path from "path";
+
+import { expect } from "chai";
+
+import WebManglerCliFileMock from "../common/file.mock";
+
+import {
+  computeStats,
+} from "../../index";
+
+suite("Compute stats", function() {
+  test("no files", function() {
+    const duration = 3.14;
+
+    const result = computeStats({
+      duration,
+      inFiles: [],
+      outFiles: [],
+    });
+
+    expect(result).to.deep.equal({
+      aggregate: {
+        changed: false,
+        changePercentage: NaN,
```

Comment:
This looks like a bug, when nothing was changed the `changePercentage` should (ideally) be `0`, not `NaN`.